cherrypy/_cperror.py

```diff
@@ -271,7 +271,7 @@ def set_response(self):
                 307: 'This resource has moved temporarily to ',
             }[status]
             msg += '<a href=%s>%s</a>.'
-            msgs = [msg % (saxutils.quoteattr(u), u) for u in self.urls]
+            msgs = [msg % (saxutils.quoteattr(u), escape_html(u)) for u in self.urls]
```

Comment:
escape_html seems to be enough to prevent XSS, and we use it for clear text inside the `<a>` tag, not for an attribute (as quoteattr would suggest), but I guess it would work as well...